clang/include/clang/Basic/TokenKinds.def

```diff
@@ -943,6 +942,15 @@ ANNOTATION(module_end)
 // into the name of a header unit.
 ANNOTATION(header_unit)
 
+// Texas Instrument
+KEYWORD(interrupt                   , KEYALL)
+KEYWORD(cregister                   , KEYALL)
+KEYWORD(reentrant                   , KEYALL)
+KEYWORD(trap                        , KEYALL)
+ALIAS("__interrupt" , interrupt     , KEYALL)
+ALIAS("__cregister", cregister      , KEYALL)
+ALIAS("__restrict" , restrict       , KEYALL)
```

Comment:
@brouxel what about `__volatile` and `__asm`?

Reply:
They were both be already present: line 671 for the volatile, line 652 for the asm.